UM/View/GL/OpenGL.py

```diff
@@ -86,7 +143,66 @@ def createShaderProgram(self, file_name):
     #                 Possible values:
     #                 - force_recreate: Ignore the cached value if set and always create a new buffer.
     def createVertexBuffer(self, mesh, **kwargs):
-        raise NotImplementedError("Should be implemented by subclasses")
+        if not kwargs.get("force_recreate", False) and hasattr(mesh, OpenGL.VertexBufferProperty):
+            return getattr(mesh, OpenGL.VertexBufferProperty)
+
+        buffer = QOpenGLBuffer(QOpenGLBuffer.VertexBuffer)
+        buffer.create()
+        buffer.bind()
+
+        buffer_size = mesh.getVertexCount() * 3 * 4 # Vertex count * number of components * sizeof(float32)
```

Comment:
Do you mean sys.getsizeof or ctypes.sizeof ? Didn't manage to get this "4" back in an easy way.